demo/src/index.js

```diff
@@ -15,30 +15,18 @@ import ContractEditor from '../../src/ContractEditor';
 
 const slateTransformer = new SlateTransformer();
 
-const templateUri = 'https://templates.accordproject.org/archives/acceptance-of-delivery@0.13.0.cta';
```

Comment:
It would be nice to keep both clauses for testing? Maybe with a bit of text in between?